best-practices.mdx

```diff
@@ -173,8 +173,16 @@ company_chalk/
 └── requirements.txt
 ```
 
-Your `.chalkignore` file should include your scripts, notebooks, and tests: anything that you are not actively using in your deployment should be put there so that
-non-deployment code does not clutter or interfere with your deployment.
+Your Chalk project must contain a project configuration file named `chalk.yaml` or `chalk.yml` in its root directory.
+You can generate this file by running [`chalk init`](/cli/init). The `chalk.yaml` file allows you to configure your
```

Comment:
`chalk init` also does a few other things, like overwrites the README and updates requirements.txt, so `chalk init` should only be used when creating a new chalk project. 

I think we can just give an example of the most basic chalk.yaml


```
project: Sandbox
environments:
    default:
        runtime: python310
        requirements: requirements.txt
```